src/Tizen.NUI/src/internal/Application.cs

```diff
@@ -1351,7 +1351,10 @@ internal bool AddIdle(SWIGTYPE_p_Dali__CallbackBase callback)
 
         public Window GetWindow()
         {
-            Window ret = new Window(Interop.Application.Application_GetWindow(swigCPtr), true);
+            Window ret = Registry.GetManagedBaseHandleFromNativePtr(Interop.Application.Application_GetWindow(swigCPtr)) as Window;
+            if(ret == null)
```

Comment:
Please use brace
if(null == ret){ 
    xxx
}
OR, you can do auto formatting in VS by "ctrl + k + d"
thank you.